src/app/pages/offline-map/offline-package-modal/offline-package-modal.component.css

```diff
@@ -1,6 +1,6 @@
 .map-container {
   padding: 16px;
-  height: calc(100vw - 32px);
+  height: 75%;
```

Comment:
Vet ikke om det er denne endringa som gjør det, men når jeg trykker på "Last ned", må jeg scrolle popup'en for å se hele "Avbryt nedlasting". Er det mulig å fikse dette? jeg har bare testet på simulator for iPad Air i Edge.
![image](https://user-images.githubusercontent.com/71138449/186896315-73bd3625-5c1e-414e-91a1-629a0a4c62a7.png)

Reply:
burde bli bra nå. har lagt til en ghost row som gjør at knappen står på samme posisjonen uansett om filene lastes ned eller ikke